### PR TITLE
Deletes empty file

### PR DESCRIPTION
**Pull request by lzimann**
why is this a thing

### CODE REVIEW

**Comment by vuonojenmustaturska** [comment]
image
lynch he

**Review by Cyberboss — DISMISSED** [review]

File: tgstation.dme [hunk 1]
```diff
@@ -367,7 +367,6 @@
 #include "code\game\gamemodes\antag_hud.dm"
 #include "code\game\gamemodes\antag_spawner.dm"
 #include "code\game\gamemodes\events.dm"
-#include "code\game\gamemodes\factions.dm"
```
Comment (Cyberboss) [inline tgstation.dme]: test

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Comment by praisenarsie** [comment]
@Xhuis todo, more like todonever

**Comment by Repukan** [comment]
Todo, more like todon't

**Comment by pubby** [comment]
# improve don't remove

**Pull request merged**